src/view/events_listeners/mustwatch_listener/mustwatch_response.py

```diff
@@ -0,0 +1,110 @@
+from typing import Dict, Tuple, Union
+
+from telebot.types import InlineKeyboardMarkup
+from telebot.types import CallbackQuery
+
+from src.view.message_text_responses import MessageTextResponse
+from src.view.buttons import Button
+
+
+class MustwatchResponse:
```

Comment:
There are lot of logic here. So looks like it's a service or manager class, not response